00007_multiply_elemets_with_constant.js

```diff
@@ -0,0 +1,14 @@
+// 7. Multiple each element of an array with a constant and print the result
+
+let inputArray = [];
+
+const multiplyArrayElementsWithConstant = (inputArray) => {
+    let result = [];
+    for (let i = 0; i < inputArray.length; i++) {
+        result[i] = inputArray[i] * 5;
+    }
+    return result
+}
+
+multiplyArrayElementsWithConstant(inputArray);
```

Comment:
What is this for?

Reply:
Actually I was meant to express how the function is being called and once I sawed your were exporting the program to check. So I thought if you were using that method, this would be useful. I can remove this from every program and I can update that. It was a confusion. I will update every code.